src/deserialize_sequence.cc

```diff
@@ -346,16 +346,28 @@ void load_step_setup_script(const std::filesystem::path& folder, Sequence& seque
     if (not std::filesystem::exists(folder))
         throw Error(gul14::cat("Folder does not exist: '", folder.string(), '\''));
 
+    std::string maintainers;
     std::string step_setup_script;
 
     auto stream = std::ifstream(folder / sequence_lua_filename);
     if (stream.good())
     {
         std::string line;
         while(std::getline(stream, line, '\n'))
+        {
+            if (gul14::starts_with(line, "-- maintainers:"))
+            {
+                // assumption: maintainers exists
+                maintainers +=
+                    ((not maintainers.empty())
+                        ? ", " + gul14::trim(line.substr(15))
+                        : gul14::trim(line.substr(15)));
```

Comment:
Maybe allowing multiple "-- maintainers:" lines and concatenating them is a bit too clever. We'll store only one line, after all. If someone edits the file by hand, it is their own fault.